wiki/Help_centre/Client/zh.md

```diff
@@ -0,0 +1,512 @@
+
+我们需要对 `cho.ppy.sh`, `osu.ppy.sh`，和 `m1.ppy.sh` 的路由追踪日志。
+
+如果要复制追踪结果，点击运行结果的底部，向上拖动，结果都变白之后即可右键复制。
+
+### 在线游玩时，我收到了服务器端的错误！ {#server-side-error}
+
+**游戏服务器可能正面临问题。**
+
+请看看 [osu! 服务器状态页面](https://status.ppy.sh/ "osu! server status")来确定是不是服务器出现了故障。
+
+**这也可能因为你使用了共享网络连接，代理服务器或者 VPN。**
+
+尝试换一个代理服务器或 VPN 服务提供商，如果不奏效，请发邮件给 [accounts@ppy.sh](mailto:accounts@ppy.sh) 来请求帮助。
+
+如果你没有用过那些东西，那么请直接请求帮助，你的连接可能被我们误封禁了。
+
+Tor 网络或通过 Tor 的连接将无法连接大部分游戏服务器。虽然我们对于 Tor 的使用不持任何立场，但是我们如此做只是为了阻止坏人使用 Tor 干坏事。
+
+### 我赞助了 osu!，但是我的用户名不是黄的！ {#white-username}
+
+**在游戏客户端当中，你自己的用户名永远是白色。其他玩家可以看到你闪闪发光的黄色名字！**
+
+这可以帮助你区分你自己和其他人的消息，目前无法禁用这个功能。
+
+## 性能 {#performance}
+
+### 我的电脑运行 osu! 会过热，怎么让它冷下来？ {#overheating}
+
+**最简单的方式就是改变你的帧数限制设置。**
+
+若要检查你目前的帧数限制，请参阅如下步骤：
+
+1. 打开 osu!。
+2. 在主页点击 `Options`，或者按下 `Ctrl` + `O`。
+3. 直接输入 `frame` 来看到你目前的帧数限制设置。
+
+如果你的设置是`无限制`，把设置更改为其他三个可能会有明显的效果。
+
+`垂直同步`会让游戏运行在你的显示器帧率下，而 `Power Saving` 会让游戏运行在显示器的两倍帧率下，`Optimal` 会让游戏运行在显示器的八倍帧率下。
+
+如果你的电脑会过热，我们建议选择`垂直同步`。
+
+你还可以了解一些控制处理器和显卡风扇转速的程序。
+
+### 玩 osu! 的时候，我的电脑自动重启！ {#reboot-overheating}
+
+**多数时候，这是因为你的电脑一个或多个部件过热了，大概率是你的显卡或者处理器。**
+
+如果要搞清楚原因，我们建议你使用检测显卡和处理器温度的程序，并且在游戏运行的时候实时关注温度值。一个很好的选择是 [Speccy](https://www.ccleaner.com/speccy)。
+
+如果你看到你的处理器和或显卡已经超过了 90°C，你的电脑就已经非常热了，并且非常接近自动重启的上限。
+
+### 我的电脑没有过热，但是游戏仍然会导致我的电脑重启！ {#reboot-faulty-device}
+
+**过热后的第二大原因就是声卡问题，第三大原因是你的外设出现问题。**
+
+旧一些的声卡驱动或软件可能会导致 osu! 出现问题，但是对大多数人来讲这不属于大问题。尝试更新你的声卡或主板驱动。
+
+你还可以试一下拔除连接到你电脑的一些 USB 外设，尤其是那些处理声音信号的设备。耳机，USB 声卡或它们连接到的接口一旦坏掉，可能会导致问题。
+
+### 还是重启！ {#reboot-nothing-helps}
+
+如果你已经试过所有方式，问题仍然出现，请联系 [accounts@ppy.sh](mailto:accounts@ppy.sh) 来请求帮助。请附上有用的信息，比如你的电脑配置，型号（如果适用）和你认为可能有助于解决问题的其他信息。
+
+### 玩 osu!mania 的时候，我不能一次性按下 n 个按钮！ {#keyboard-ghosting}
+
+**这是因为大部分键盘都存在的*按键冲突*问题。**
+
+按键冲突是个有一点复杂的问题，本质是你的键盘制作工艺上的缺陷，导致它不能发送一次性按下多个键的信号。
+
+大部分的新键盘可以一次性按下**6**个按键，顺序和按什么不限。
+
+你可以用 [drakeirving 的键盘冲突小工具](https://drakeirving.github.io/MultiKeyDisplay)来检查自己的键盘一次性可以正常按下多少个键。
+
+虽然 osu!mania 的 Ranking Criteria 已经规定谱面不能出现同时按下6个键以上的配置，但如果你的键盘比较老旧，按键冲突的问题可能还是会影响到你。
+
+治标治本的解决方案只有购买一个能同时处理更多按键的新键盘。如果那样的话，你可能可以关注*全键无冲*的键盘。这种键盘基本不会有按键冲突的问题，但是一般比较昂贵。
+
+### 玩 osu! 的时候，我的光标疯跑！ {#crazy-cursor}
+
+**最常见的原因是你的游戏内鼠标灵敏度设置的太高了！**
+
+乱跳，诡异的鼠标移动是最典型的例子，尤其是当你使用数位板的时候。
+
+大部分情况下，把游戏内的灵敏度更改为 1.0x（默认设置）会解决这个问题，但是可能会导致你的鼠标移动比较缓慢。我们建议你打开游戏中的 `Raw Input` 来让 osu! 直接通过设备驱动来调整灵敏度。
+
+如果你使用鼠标，你还可以尝试更改你的鼠标 *DPI* 设置。更高的 DPI 设置能让游戏收到更准确的定位，同时可能会平滑掉迟缓的感觉。
+
+我们还建议你检查自己的外设是否足够的干净。如果有头发或者灰尘挡住鼠标的激光传感器，可能会导致鼠标移动很怪。
+
+#### 启用 Raw Input {#raw-input}
+
+**Raw input 会让游戏直接与你的外设通信，获得更加直接且未经处理的移动数据，这通常会让设备输出的精度提升。**
+
+若要打开 Raw Input，请参阅以下步骤：
+
+1. 打开 osu!。
+2. 点击主页的 `Options`，或者按下 `Ctrl` + `O`。
+3. 直接输入 `raw`，即可跳转到对应的设置区域。
+4. 点击 `Raw input` 来将它打开。
+
+### 我使用数位板，我感觉我的光标移动太慢了！ {#slow-cursor}
+
+**你可以设置更小的数位板映射区域来解决问题。**
+
+在大多数 Wacom 数位板上，你可以在 `Wacom 数位板属性`的`映射`标签页中完成操作。
+
+![Wacom Tablet Properties screen showing Mapping options](img/tablet-area.png "Interface for changing tablet area on Wacom devices")
```

Comment:
```suggestion
![Wacom 数位板属性屏幕显示映射选项](img/tablet-area.png "Interface for changing tablet area on Wacom devices")
```

Reply:
the tooltip has to be translated as well

Reply:
```suggestion
![Wacom 数位板属性屏幕显示映射选项](img/tablet-area.png "Wacom 设备更改数位板映射区域的界面")
```